front_end/src/app/(main)/questions/[id]/components/forecast_maker/binary_slider.tsx

```diff
@@ -74,17 +88,48 @@ const BinarySlider: FC<Props> = ({
           marks={
             communityForecast
               ? {
+                  // Pass the isNear prop
                   [communityForecast * 100]: (
-                    <MarkArrow value={communityForecast} />
+                    <MarkArrow
+                      value={communityForecast}
+                      isNear={isNearCommunityForecast}
+                    />
                   ),
                 }
               : undefined
           }
           disabled={disabled}
           styles={disabled ? { handle: { cursor: "default" } } : {}}
+          showValue
         />
+        {forecast !== null && helperDisplay && (
+          <div
+            className="absolute flex flex-col items-center"
+            style={{
+              left: `${((forecast - BINARY_MIN_VALUE) / (BINARY_MAX_VALUE - BINARY_MIN_VALUE)) * 100}%`,
+              transform: "translateX(-50%)",
+            }}
+          >
+            <div className="w-full self-center text-center text-orange-700/50 dark:text-orange-300/50">
+              <FontAwesomeIcon icon={faChevronUp} />
+            </div>
+            <span className="mt-[-4px] block text-orange-700 dark:text-orange-300">
+              Me
+            </span>
+          </div>
+        )}
+        {/* {helperDisplay && ( // Conditionally render the "Very Unlikely" and "Very Likely" labels based on helperDisplay prop
+          <>
+              <div className="absolute left-0 top-4 mt-2 text-xs text-gray-500 z-1">
+                Very Unlikely
+              </div>
+              <div className="absolute right-0 top-4 mt-2 text-xs text-gray-500 z-1">
+                Very Likely
+              </div>
+          </>
+        )} */}
       </div>
-      <div className="mb-3 block text-center">
+      {/* <div className="mb-3 block text-center">
```

Comment:
The ForecastTextInpout seems to remain commented here. Not easy to see if this is intentional, but just flagging it to make sure it's not a mistake.

Reply:
Yes, with the new design we lost the ability to enter text input for binary sliders.